massa-graph/src/block_graph.rs

```diff
@@ -591,7 +591,7 @@ impl BlockGraph {
     pub fn export_bootstrap_graph(&self) -> Result<BootstrapableGraph> {
         let mut required_final_blocks: Set<_> = self.list_required_active_blocks()?;
         required_final_blocks.retain(|b_id| {
-            if let Some(BlockStatus::Active(a_block)) = self.block_statuses.get(&b_id) {
+            if let Some(BlockStatus::Active(a_block)) = self.block_statuses.get(b_id) {
```

Comment:
fixed this here already https://github.com/massalabs/massa/pull/2765

Reply:
is there a conflict ? I think it's ok, we can just merge this one

Reply:
Yes it's ok